content/02.introduction.md

```diff
@@ -115,11 +115,39 @@ Respiratory failure remains the leading cause of death for COVID-19 patients [@d
+The first study, which is by Blanco-Melo et al., performed an in-depth analysis of the transcriptional response to SARS-CoV-2 in comparison to other respiratory viruses, including MERS-CoV, SARS-CoV-1, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV). 
+They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
+As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with a vector expressing mCherry or ACE2 (A549-ACE2). 
+All infections were performed at a high multiplicity of infections (MOI, 2–5), except A549-ACE2 cells were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).
+Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed after 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
+The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients. 
+Additionally, transcriptional response to IAV in NHBE cells and nasal washes of ferrets was measured.
+Differential expression (DE) analysis was carried out between infected and respective mock (control) cell condition.
+DE results revealed that the transcriptional response of the cells with efficient SARS-CoV-2 replication is significantly different from the host response of all other viruses tested. 
+A unique proinflammatory cytokine signature is present in both high and low multiplicity of infections (MOI). 
+However, in contrast to high-MOI, low-MOI SARS-CoV-2-infected A549-ACE2 cells showed no significant IFN-I or IFN-III expression, suggesting that IFN induction is MOI dependent. 
+Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, they found significant enrichment of chemokine signaling but no significant induction of IFN-I or IFN-III. 
+Together, these results suggest that in contrast to common respiratory viruses, SARS-CoV-2 induces limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated genes (ISG) response. 
+However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions [@doi:10.1016/j.cell.2020.04.026]. 
+This speculation was further supported by a recent study [@doi:10.1101/2020.05.11.088179], showing that the SARS-CoV-2 ORF3b gene, whose length is considerably smaller as compared to the SARS-CoV-1 ORF3b gene, suppress IFNB1 promoter activity (IFN-I induction) more efficiently than SARS-CoV-1.
```

Comment:
```suggestion
This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV-1 _ORF3b_ gene, which is considerably shorter.
```

Reply:
@rando2 Thanks. I will finish this part during next weekend. "which is considerably shorter" 
should be 
"which is considerably longer".